src/sage/schemes/elliptic_curves/mod_poly.py

```diff
@@ -87,6 +87,21 @@ def classical_modular_polynomial(l, j=None):
         sage: Y = polygen(parent(j), 'Y')
         sage: classical_modular_polynomial(l,j) == classical_modular_polynomial(l)(j,Y)
         True
+        sage: p = 2^216 * 3^137 - 1
+        sage: F.<i> = GF(p^2, modulus=[1,0,1])
+        sage: l = random_prime(50)
```

Comment:
you could use the syntax `GF((p,2),...)` to avoid factoring.

Reply:
Thanks, I didn't know this constructor!